### DISCUSSION TITLE
好消息：10抽出金 壞消息：歪了

### DISCUSSION THREAD

**Discussion by Andrea0126** [share121/inter-knot#5397] [绝区零]
![Screenshot_2024-09-25-14-59-33-278_com HoYoverse Nap](https://github.com/user-attachments/assets/8b7901b5-d801-467c-8644-63b0c9e4c3ef)
不是你歪也給我歪狼哥啊😭
༼⁠;⁠´⁠༎ຶ⁠ ⁠۝ ⁠༎ຶ⁠༽

**Comment by Robin-txdy** [comment]
我也歪了丽娜<img src="https://bbs-static.miyoushe.com/static/2023/05/23/a2fa805a0a782cdb8ab44ddaf8d5343d_962898009752203374.png" alt="阿姬倒地了04" width="75" >

**Comment by Corona618** [comment]
壞消息：我也歪麗娜
好消息：我雙黃
![Screenshot_20240925-110514](https://github.com/user-attachments/assets/9e0c6b1b-32d4-4edc-8bc9-ad47560dffd7)

**Reply by Loxia1121** [reply]
你不许玩绝区零

**Comment by aino2333** [comment]
我还没出呢，我还要50多发呢<img src="https://img-static.mihoyo.com/communityweb/upload/7bd889870907d2d96673d09afb53cf63.png" alt="三月七_哭" width="75" >
